cohydra/channel/channel.py

```diff
@@ -18,22 +18,27 @@ class Channel:
     ----------
     network : :class:`.Network`
         The network this channel belongs to.
+    channel_name : str
+        The name of the channel
     nodes : list of :class:`.Node`
         The nodes to connect on a physical channel.
     """
 
-    def __init__(self, network, nodes):
+    def __init__(self, network, channel_name, nodes):
```

Comment:
Is it now a requirement that every channel has a name? Can there be 'anonymous' channels, or can we at least infer a default value from the network and nodes?

Reply:
Somehow yes. But (pointing at your other comment) we can generate a name for it. But this is the wrong place for that - `create_channel` fits better in my opinion.